bitcoin-internals/README.md

```diff
@@ -0,0 +1,7 @@
+Rust Bitcoin Internal
+=====================
+
+This crate is only meant to be used internally by crates in the
+[rust-bitcoin](https://github.com/rust-bitcoin) ecosystem.
+
+This crate will never be stabalized, depend on it at your own risk.
```

Comment:
s/stabalized/stabilized/

Reply:
Thanks, this finally prompted me to set my dictionary to "american" instead of always ignoring the spelling mistakes on US spelled words (British use an 's').

Reply:
(A little bit of me just died :) - "merica!"